### PR TITLE
@broskoski => Merchandisable artists aggregation results

### PR DESCRIPTION
**Pull request by mzikherman**
screen shot 2016-07-14 at 4 04 39 pm

For use in: 
screen shot 2016-07-14 at 11 41 37 am

This will just be an aggregation, where we also receive the actual artists in order of max merch score.

### CODE REVIEW

File: schema/filter_artworks.js [hunk 1]
```diff
@@ -28,6 +30,16 @@ export const FilterArtworksType = new GraphQLObjectType({
...
       resolve: ({ aggregations }) => aggregations.followed_artists.value,
     },
+    merchandisable_artists: {
+      type: new GraphQLList(Artist.type),
+      description: 'Returns a list of merchandisable artists (sorted by merch score), if the aggregation was requested.',
+      resolve: ({ aggregations }) => {
+        if (!isExisty(aggregations.merchandisable_artists)) {
+          return null;
+        }
+        return gravity(`artists`, { ids: keys(aggregations.merchandisable_artists) });
```
Comment (mzikherman) [inline schema/filter_artworks.js]: https://github.com/artsy/gravity/pull/10228 fetch all artists by id, maintaining the order

File: schema/aggregations/filter_artworks_aggregation.js [hunk 2]
```diff
@@ -42,6 +42,9 @@ export const ArtworksAggregation = new GraphQLEnumType({
     FOLLOWED_ARTISTS: {
       value: 'followed_artists',
     },
+    MERCHANDISABLE_ARTISTS: {
```
Comment (mzikherman) [inline schema/aggregations/filter_artworks_aggregation.js]: Done in https://github.com/artsy/gravity/pull/10227

File: schema/filter_artworks.js (hunk 1 above)
Comment (broskoski) [inline schema/filter_artworks.js]: ✅

**Comment by broskoski** [comment]
Looks great but come on Semaphore

**Comment by mzikherman** [comment]
💚

**Pull request merged**